db/hints/sync_point.cc

```diff
@@ -26,20 +26,20 @@ namespace hints {
 //
 // Format V1 (encoded in base64):
 //   uint8_t 0x01 - version of format
-//   sync_point_v1 - encoded using IDL
+//   sync_point_v1_or_v2 - encoded using IDL
 //
 // Format V2 (encoded in base64):
 //   uint8_t 0x02 - version of format
-//   sync_point_v1 - encoded using IDL
+//   sync_point_v1_or_v2 - encoded using IDL
```

Comment:
Comment about the first commit's title:

> refactor sync point version types 

If you can, please make the titles more specific. For example, here you could do:

`db: hints: rename structures with _v1 suffix to have _v1_v2 suffix`

We usually use a convention where we prefix the titles with the name of the component (look at the git log for inspiration).